bin/propolis-utils/src/bin/cpuid-gen.rs

```diff
@@ -261,15 +262,116 @@ fn print_toml(results: &BTreeMap<CpuidKey, Cpuid>) {
     }
 }
 
+fn print_json(results: &BTreeMap<CpuidKey, Cpuid>) {
+    use propolis_api_types::instance_spec::components::board::{
+        Cpuid, CpuidEntry,
+    };
+
+    let vendor = {
+        use propolis_api_types::instance_spec::{CpuidValues, CpuidVendor};
+        match results.get(&CpuidKey::Leaf(0)) {
+            None => {
+                eprintln!("no result for leaf 0, selecting default vendor");
```

Comment:
super nitpicky, but perhaps this should actually say who the default vendor is?
```suggestion
                eprintln!("no result for leaf 0, selecting default vendor (AMD)");
```
or similar?

Reply:
Done in 4aaff74.